js/modules/loki_app_dot_net_api.js

```diff
@@ -34,6 +48,123 @@ const snodeHttpsAgent = new https.Agent({
 
 const timeoutDelay = ms => new Promise(resolve => setTimeout(resolve, ms));
 
+let adnOnionRequestCounter = 0;
+const sendViaOnion = async (srvPubKey, url, pFetchOptions, options = {}) => {
+  if (!srvPubKey) {
+    log.error(
+      'loki_app_dot_net:::sendViaOnion - called without a server public key'
+    );
+    return {};
+  }
+
+  // set retry count
+  let adnOnionRequestCount;
+  if (options.retry === undefined) {
+    // eslint-disable-next-line no-param-reassign
+    options.retry = 0;
+    adnOnionRequestCounter += 1; // increment counter
+    adnOnionRequestCount = 0 + adnOnionRequestCounter; // copy value
```

Comment:
What is actually the difference between the two? Is `options.counter` used to persist the same value across multiple attempts? Then it probably needs a better name.

Reply:
> Is options.counter used to persist the same value across multiple attempts? 
Yes

> I would like to unify reqIdx and lsrpcIdx though.
Ok, done, will push in a sec.